### PR TITLE
Finished with Sign In and Sign Up views

### PR DESCRIPTION
**Pull request by Misha7776**

### CODE REVIEW

**Comment by Kmalynss** [comment]
Okay, so do we rly need all those files in the project directory? Because according to https://github.com/twbs/bootstrap-sass those are available through the pipeline

**Comment by Misha7776** [comment]
Today I will solve the question.

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,7 @@
  *= require_tree .
  *= require_self
  */
+
+
+
+
```
Comment (Kmalynss) [inline app/assets/stylesheets/application.css]: Is there any reason for this empty lines to be here?

File: app/assets/stylesheets/css/app.css [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/*@import url('bootstrap-extended.css');*/
+
```
Comment (Kmalynss) [inline app/assets/stylesheets/css/app.css]: Is this empty line needed here?

File: app/assets/stylesheets/custome.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@import "bootstrap-sprockets";
```
Comment (Kmalynss) [inline app/assets/stylesheets/custome.scss]: may be *custom*

**Comment by Misha7776** [comment]
What you mean with custom.css , should i put it into application.css?

File: app/assets/stylesheets/custome.scss (hunk 3 above)
Comment (Misha7776) [reply app/assets/stylesheets/custome.scss]: Don`t understand what to improve here. What exactly i should do here?

Comment (DimaSmagulov) [reply app/assets/stylesheets/custome.scss]: change file name)

Comment (Misha7776) [reply app/assets/stylesheets/custome.scss]: LOL, sorry i have understand)))) Willl rename)

**Comment by Kmalynss** [comment]
Do we still need these files:
 - app/assets/stylesheets/css/bootstrap.min.css
 - app/assets/stylesheets/css/bootstrap.css
 - app/assets/stylesheets/css/bootstrap-extended.min.css
 - app/assets/stylesheets/css/bootstrap-extended.css
 - app/assets/stylesheets/css/app.min.css
?

**Comment by Misha7776** [comment]
Yes, we need this files because they provide css for our pattern.
I could not find a way to do without them.

**Comment by Kmalynss** [comment]
Okay, than what about this PR: https://github.com/Ruby-IF081/heroes_team_app/pull/26/files ?
Bootstrap is also used there but there is nonsuch files.

File: app/views/layouts/_footer.html.slim [hunk 4]
```diff
@@ -0,0 +1,10 @@
+footer.footer.fixed-bottom.footer-dark.navbar-shadow
+  p.clearfix.blue-grey.lighten-2.text-sm-center.mb-0.px-2
+    span.float-md-left.d-block.d-md-inline-block
+      | Copyright © 2017
```
Comment (igorkasyanchuk) [inline app/views/layouts/_footer.html.slim]: don't hardcode year. use ruby.

File: app/views/layouts/_footer.html.slim [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+      a.text-bold-800.grey.darken-2[href="https://softserveinc.com/en-us/" target="_blank"]
+        | Soft Serve
```
Comment (igorkasyanchuk) [inline app/views/layouts/_footer.html.slim]: SoftServe

**Pull request merged**